notes/returned-mail-received.vbs

```diff
@@ -376,51 +410,107 @@ IF CRF_checkbox = CHECKED THEN pending_verifs = pending_verifs & "Change Report
 pending_verifs = trim(pending_verifs) 	'takes the last comma off of pending_verifs
 IF right(pending_verifs, 1) = "," THEN pending_verifs = left(pending_verifs, len(pending_verifs) - 1)
 '----------------------------------------------------------------------------------------------------TIKL
+due_date = dateadd("d", 10, date_requested)	'defaults the due date for all verifications at 10 days
```

Comment:
date_requested is not always defined - not in every option. so due_date will sometimes create the date 1/9/1900. due_date is used in a lot of the options

Reply:
made an edit box